torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1637,6 +1637,23 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
            sample_inputs_func=sample_inputs_fliplr_flipud,
            test_inplace_grad=False,
            supports_tensor_out=False),
+    UnaryUfuncInfo('frexp',
+                   op=torch.frexp,
+                   ref=np_unary_ufunc_integer_promotion_wrapper(np.frexp),
+                   dtypesIfCPU=all_types_and(torch.bool, torch.half),
+                   dtypesIfCUDA=floating_types_and(torch.half),
+                   # skip testing torch.frexp as it is not supported by ROCM platform yet
+                   dtypesIfROCM=_dispatch_dtypes(()),
+                   test_inplace_grad=False,
+                   safe_casts_outputs=True,
+                   skips=(
+                       # Reference: https://github.com/pytorch/pytorch/pull/51097
+                       # numpy returns exponents as integral arrays, while torch.frexp promotes integral inputs
+                       # to floating types
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics'),
```

Comment:
The reason to skip this test is https://github.com/pytorch/pytorch/pull/51097#issuecomment-774948814